ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ExpressionTypeManager.java

```diff
@@ -189,13 +189,13 @@ public Void visitLambdaVariable(
         final LambdaVariable node, final Context context
     ) {
       context.setSqlType(
-          context.getLambdaSqlTypeMapping().get(node.getValue())
+          context.getLambdaSqlTypeMapping().get(node.getLambdaCharacter())
```

Comment:
nit: this is for line 130 above. I'm wondering if we should not provide two overloaded `getExpressionSqlType` and just enforce all callers to explicitly create `emptyMap` if necessary, to avoid potential bugs like this?

Reply:
Yeah that sounds reasonable. I'll do it in a follow up at some point.